### DISCUSSION TITLE
How to analyze the response to an operation?

### DISCUSSION THREAD

**Discussion by iPlayC** [mitre/caldera#2783] [Q&A]
Hello everyone,

I have a question, I'd like to test my security solution with Caldera, but I don't know how to understand and analyze the results that come out after launching my operation with my different adversaries.

Thank you for your answers.

Sincerely 

iPlayC

**Comment by clenk** [comment]
After running an operation you can download the operation as JSON, but it's not the most human-readable. Have you seen the `debrief` plugin yet? It will give you some easier to read graphs and tables.
